examples/Hermitian_Phasing.py

```diff
@@ -80,20 +80,21 @@
         logger.info(strm("Rough center is:", rough_center))
         data.setaxis("t2", lambda x: x - rough_center).register_axis({"t2": 0})
         data.ft("t2")
-        mysgn = select_pathway(data, signal_pathway).C.real.sum("t2").run(np.sign)
-        data *= mysgn
         data.ift("t2")
+        # {{{ this is not correct -- you want to use the zeroth order phasing
+        # routine to determine the zeroth order phase, since it allows for
+        # negative data this will flip the negative data to positive
         ph0 = select_pathway(data, signal_pathway)["t2":0]
         ph0 /= abs(ph0)
         data /= ph0
+        # }}}
```

Comment:
resolve this here, first, since it will be easier --- feed `select_pathway(data, signal_pathway)["t2":0]` to the zeroth order routine, and show the diagnostic plot.

the correlation example has the same error, and will have the same solution

Reply:
okay so now this PR is fixing both the correlation alignment example and the phasing example

Reply:
well, the should have been fixed in the other PR before it got merged, but something funky happened there

Reply:
fixed both accordingly